### ISSUE TITLE
[Warrior] [Fury] Offhand Hit Rating Incorrectly Calculated

### ISSUE THREAD

**Issue by Furtavious** [dalaranwow/dalaran-wow#5714]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The warrior's special *yellow* attacks require the expected 8% hit rating to never miss, while the white attacks have the expected required 27% hit rating needed in order to never miss. When having heroic strike queued the following off hand white attack has the properties of a normal white attack, requiring 27% hit rating to not miss and still allows for glancing blows. 

**Expected behaviour**: Tell us what should happen instead

When duel wielding with heroic strike queued, the following off hand white attack was counted as a special attack on the combat table, so it only requires 8% hit rating to not miss (this also obviously shoves glancing blows off the table, as it is counted as a special attack)

**Steps to reproduce the problem**:

1.  Attack a lvl 83 boss target with 8% hit (5% with 3 points in precision)
2. Queue Heroic strike 
3.  Observe glancing blows/misses from the off hand attacks

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

https://web.archive.org/web/20130209181158/http://elitistjerks.com/f47/t37804-warrior_fury_compendium/#Understanding_the_Hit_table 

This article describes in detail the right way the combat table should act. To summarize, the meat of the issue can be found under the section "Understanding the Hit table" subsection "Effect of Heroic Strike".

I'll include the paragraph with the main part of this issue.

"When you use Heroic Strike, it turns your next main hand white ability into a yellow one. Therefore, instead of your normal melee attacks using the single-roll white table, they use the two-roll yellow table. This removes the glancing blow chance, reduces the base miss rate, and increases the crit bonus damage of the swing if it were to crit via the Impale talent. Heroic strike also adds 495 damage to the swing. Turning these white swings into more powerful yellow swings is a very solid boost in dps, however the rage cost of doing this is high."

You can also farther verify this issue by looking at how fury warrior dps is calculated via simulation programs, such as Landsoul's, which as a specific part that mentions this effect. 

The following is a picture of part of landsoul's crafting excel spreadsheet, which allows for the option to add HS + 0 Miss bug

https://gyazo.com/53b8e852aa7458d7c9fb34422bc5149b

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Furtavious** [comment]
I feel like this is a pretty significant class bug to go 2 months with no replies. Is there something more that needs to be done to farther proof or? This is around a 10% damage difference for warriors but not high on the priority list?

**Comment by Aridam** [comment]
Bump.

This is a critical bug for Fury Warriors as an example, as pre-Cata, this was a fundamental mechanic for them and one of the main reasons for their stupidly-high DpS -- by spamming Heroic Strike/Cleave, their Off-Hand inherits the removal of the Dual Wield penalties and treating the Off-Hand as an Ability hit (despite it still being shown as "white Damage"), enabling Fury Warriors to avoid Misses and Glances as much as possible, maximizing their Damage/DpS/Rage gains with their Off-Hand. By being Hit-capped, Expertise-capped, standing behind their Targets to remove Parry Chance, Fury Warriors could keep spamming HS/Cleave and force their Off-Hand to _always_ hit, _never_ Miss, _never_ Glance.

Not the case here.

With this bug, all of that is gone, and it can further force Fury Warriors to unqueue Heroic Strike/Cleave at times to make up for this and gain back some Rage, allowing their Main Hand to also Miss/Glance and just go further-down the rabbit hole.

The provided proof is enough, so here's additional video evidence of the bug on Dalaran-WoW;
https://www.youtube.com/watch?v=4IXTCJHtVto
Level 60 Dummy. Due to the Dual Wield penalty on Hit, I can get Misses. I proceed to spam Heroic Strike, but the result is the same -- my Off-Hand still randomly gets Misses, even though it shouldn't.

https://www.youtube.com/watch?v=6lrrQ6Zp0Vo
Level ?? Heroic Dummy. Glances can happen. I once again spam Heroic Strike to force the Off-Hand to never Glance, yet it still does despite that.
